src/util.h

```diff
@@ -34,6 +34,22 @@
  * util.h -- internal definitions for util module
  */
 
+#include <stdbool.h>
+
+/*
+ * Compiler aligns __m128, __m128d and __m128i local and global
+ * data 16 byte boundaries. MOVNT values below represent this alignment.
+ * MOVNTx is the assembly language command for moving data using
+ * non-temporal location.
+ */
+
+#define	MOVNT_SIZE	16
```

Comment:
Why are these defines here, while used in *pmem.c* only?
*pmem.h* would be more appropriate, I suppose.